### PR TITLE
[1LP][RFR] Change import lines order to be alphabetical.

### PR DESCRIPTION
**Pull request by rhrazdil**
Fixing imports order to be alphabetical in integration module.

### CODE REVIEW

File: cfme/tests/infrastructure/test_advanced_search_host.py [hunk 1]
```diff
@@ -2,10 +2,10 @@
 """This testing module tests the behaviour of the search box in the Hosts section"""
 import fauxfactory
 import pytest
-from itertools import dropwhile
-from widgetastic.exceptions import NoSuchElementException
 
 from cfme.utils.appliance.implementations.ui import navigate_to
+from itertools import dropwhile
+from widgetastic.exceptions import NoSuchElementException
```
Comment (jkrocil) [inline cfme/tests/infrastructure/test_advanced_search_host.py]: [Developer Guidelines](http://cfme-tests.readthedocs.org/guides/dev_guide.html)

> In addition to being broken up into the **three sections** of standard library, third-party, and the local application, imports should be sorted alphabetically.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/infrastructure/test_webmks_console.py [hunk 2]
```diff
@@ -10,8 +10,8 @@
 from cfme.utils.conf import credentials
 from cfme.utils.log import logger
 from cfme.utils.providers import ProviderFilter
-from wait_for import wait_for
 from markers.env_markers.provider import providers
+from wait_for import wait_for
```
Comment (psav) [inline cfme/tests/infrastructure/test_webmks_console.py]: **REQUIRED**: ```wait_for``` is an external tool now and hence should be with the others......

**Comment by psav** [comment]
I think I will just run these through pyCharm to keep is consistent

**Comment by psav** [comment]
#6828

**Pull request closed without merging**